lte/gateway/python/magma/enodebd/tests/baicells_qrtb_tests.py

```diff
@@ -801,7 +802,7 @@ class BaicellsQRTBFirmwareUpgradeDownloadTests(EnodebHandlerTestCase):
     }
 
     # configs which should not lead to firmware upgrade download flow
-    config_empty = {'firmwares': {}, 'enbs': {}, 'models': {}}
+    config_empty: Dict[str, Dict[Any, Any]] = {'firmwares': {}, 'enbs': {}, 'models': {}}
```

Comment:
Looks more like a `Dict[str, Any]`?

Reply:
Interpreted `{}` to be a dict. But I that might be to strict. Changing it.